src/components/views/context_menus/RoomGeneralContextMenu.tsx

```diff
@@ -45,13 +45,60 @@ import { useSettingValue } from "../../../hooks/useSettings";
 
 export interface RoomGeneralContextMenuProps extends IContextMenuProps {
     room: Room;
+    /**
+     * Called when the 'favourite' option is clicked, after the menu has processed
+     * whatever caused the click in the normal way.
+     * @param event The event that caused the click
+     */
     onPostFavoriteClick?: (event: ButtonEvent) => void;
+    /**
+     * Called when the 'low priority'' option is clicked, after the menu has processed
+     * whatever caused the click in the normal way.
+     * @param event The event that caused the click
+     */
     onPostLowPriorityClick?: (event: ButtonEvent) => void;
+    /**
+     * Called when the invite option is clicked, after the menu has processed
```

Comment:
nit: consistent use of quotes would be nice

```suggestion
     * Called when the 'invite' option is clicked, after the menu has processed
```

likewise below